proxy/src/control/destination/background.rs

```diff
@@ -72,6 +88,18 @@ struct DestinationSet<T: HttpService<ResponseBody = RecvBody>> {
     responders: Vec<Responder>,
 }
 
+/// The state needed to bind a new controller client stack.
+///
+/// This implements `watch_service::Rebind`, so it can be used to rebind the
+/// client when the TLS client configuration changes.
```

Comment:
No need to say that it implements `Rebind` in a comment since later in the code we'll have `impl Rebind for BindClient`.